src/Pos/Wallet/Web/Api.hs

```diff
@@ -257,20 +256,13 @@ type UpdateTx =
     :> ReqBody '[JSON] CTxMeta
     :> WRes Post ()
 
-type GetHistory =
```

Comment:
@Pastafarianist anyway, if it does, you can now rewrite method `getHistory` in bridge (`src/Daedalus/BackendApi.purs:181`) in such a way that it uses `searchHistory` api method without search query.